packages/launchdarkly-adapter/modules/adapter/adapter.ts

```diff
@@ -223,6 +235,7 @@ const configure = ({
     if (subscribeToFlagChanges && flagsFromSdk)
       setupFlagSubcription({
         flagsFromSdk,
+        flagsUpdateDelayMs,
```

Comment:
This would be `<ConfigureFlopflip adapter={ldAdapter} adapterArgs={{ ..., flagsUpdateDelayMs: 1000 }} />